firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/FirebaseCrashlytics.java

```diff
@@ -161,7 +161,7 @@
 
     final String googleAppId = app.getOptions().getApplicationId();
     final String mappingFileId = CommonUtils.getMappingFileId(context);
-    Logger.getLogger().d("Mapping file ID is: " + mappingFileId);
```

Comment:
Strongly suggest this line remain DEBUG since it is critical to many support cases.

Reply:
👍